docs/docusaurus.config.js

```diff
@@ -0,0 +1,117 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'UrTurn',
+  tagline: 'Quickly build multiplayer games - only four functions!',
+  url: 'https://docs.urturn.app',
+  baseUrl: '/',
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
+  // favicon: 'img/favicon.ico', TODO: replace with turn based games logo
+  organizationName: 'turnbasedgames',
+  projectName: 'urturn',
```

Comment:
I changed the `projectName` from `turnbasedgames.github.io` to `urturn`. This allows the deploy command, ` lerna run deploy --scope=@urturn/docs`, to work.